src/views/Main/index.js

```diff
@@ -1,16 +1,13 @@
 import { connectAuthProvider } from '@terralego/core/modules/Auth';
-import { withTranslation } from 'react-i18next';
 import { connectAppProvider } from '../../components/AppProvider';
 import { withLocale } from '../../components/Locale';
 import Main from './Main';
 
 export default
-withTranslation()(
-  withLocale(
-    connectAppProvider('env', 'errorSettings')(
-      connectAuthProvider('authenticated')(
-        Main,
-      ),
+withLocale(
+  connectAppProvider('env', 'errorSettings')(
+    connectAuthProvider('authenticated')(
+      Main,
     ),
   ),
 );
```

Comment:
Couldn't we take the opportunity to use a compose function there?
```js
export default compose(
  withLocale,
  connectAppProvider('env', 'errorSettings'),
  connectAuthProvider('authenticated'),
)(Main);
```

Reply:
Done 👍